queries.sql

```diff
@@ -53,3 +53,25 @@ select species, min(weight_kg), max(weight_kg) from animals group by species;
 
 --What is the average number of escape attempts per animal type of those born between 1990 and 2000?
 select species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '01-01-1990' AND '31-12-2000' GROUP BY species;
+
+--Project 3
+--What animals belong to Melody Pond?
+SELECT animals.name FROM owners JOIN animals ON owners.id = animals.owner_id WHERE full_name = 'Melody Pond';
+
+--List of all animals that are pokemon (their type is Pokemon).
+SELECT animals.name FROM species JOIN animals ON species.id = animals.species_id WHERE species.name = 'Pokemon';
+
+--List all owners and their animals, remember to include those that don't own any animal.
+SELECT owners.full_name, animals.name FROM owners LEFT JOIN animals ON owners.id = animals.owner_id;
+
+--How many animals are there per species?
+SELECT species.name, count(*) FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.name;
+
+--List all Digimon owned by Jennifer Orwell.
+SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name='Jennifer Orwell';
```

Comment:
- [ Required ] I think to fetch all `Digimon` species owned by `Jennifer Orwell` it would be better to refactor this query as shown below, currently it fetches all animals owned by `Jennifer Orwell` including those that are not of `Digimon` species.
```SQL
 SELECT animals.name FROM animals LEFT JOIN owners ON animals.owner_id = owners.id 
 LEFT JOIN species ON animals.species_id = species.id WHERE owners.full_name='Jennifer Orwell' 
 AND species.name = 'Digimon';
```

Reply:
Thank you very much, there was my mistake! Let me fix it.

Reply:
![image](https://user-images.githubusercontent.com/106581139/214547207-30a5cae9-2d46-4470-84bf-241937dbd10a.png)